### ISSUE TITLE
Is it possible to open source the RoCEv2 w/ DCQCN implementation? 

### ISSUE THREAD

**Issue by YangZhou1997** [minmit/tonic#1]
I saw the tonic paper do describe this implementation. 

Thanks! 
Yang

**Comment by minmit** [comment]
Yes! We are restarting the porting from the dev repo to the public repo in a few weeks, and this will be part of that too.

**Comment by taekyounghan** [comment]
I am looking forward to the disclosure of rate-based implementation. When will it be revealed?

**Comment by minmit** [comment]
Our previous efforts got delayed but our next goal is for it to be out in the spring. In the meantime, appendix B in the paper has some information about the implementation of the rate limiters and I'll be happy to answer questions about other details.

**Comment by taekyounghan** [comment]
> Our previous efforts got delayed but our next goal is for it to be out in the spring. In the meantime, appendix B in the paper has some information about the implementation of the rate limiters and I'll be happy to answer questions about other details.

Thank you for your kind reply!! :-)

**Comment by mabellato** [comment]
> Our previous efforts got delayed but our next goal is for it to be out in the spring. In the meantime, appendix B in the paper has some information about the implementation of the rate limiters and I'll be happy to answer questions about other details.

I read the paper and me too was wondering if you can disclose the ROCEV2 with DCQCN implementation. Any follow-up on this ? thanks
